gr-digital/python/digital/qam.py

```diff
@@ -29,16 +42,13 @@
 # coding is used within but not between each quadrant.
 _def_mod_code = mod_codes.NO_CODE
 
-
 def is_power_of_four(x):
     v = log(x) / log(4)
     return int(v) == v
 
-
 def get_bit(x, n):
     """ Get the n'th bit of integer x (from little end)."""
-    return (x & (0x01 << n)) >> n
-
+    return (x&(0x01 << n)) >> n
```

Comment:
I think Pylint prefers the original version.

Reply:
In fact, can you please remove all whitespace changes from this diff? They make our Pylint scores worse, and should not be mixed up with a new class.

Reply:
I'm not really sure what you are referring to here. There are 2 blank lines between defs in the current code in the master branch

Reply:
Nevermind, I believe I've fixed them all now.